ubuntu/gpu/cuda-11/conda/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM panchalhp/gpu-base:cuda11
```

Comment:
`databricksruntime` to replace `panchalhp`

Reply:
Yep, I'll do that once I have push access to databricksruntime. I'll have to chat with Evan about getting permission. Will change this before merging it :)